### PR TITLE
Update scheduler-equivalence-class.md

### PR DESCRIPTION
**Pull request by AdamDang**
Fix some typos:
binded->bound
we does not -> we do not

### CODE REVIEW

File: contributors/design-proposals/scheduling/scheduler-equivalence-class.md [hunk 1]
```diff
@@ -229,14 +229,14 @@ Please note with the change of predicates in subsequent development, this doc wi
...
-    - No `MatchInterPodAffinity`: the scheduler will make sure newly binded pod will not break the existing inter pod affinity. So we does not need to invalidate MatchInterPodAffinity when pod added. But when a pod is deleted, existing inter pod affinity may become invalid. (e.g. this pod was preferred by some else, or vice versa).
+    - No `MatchInterPodAffinity`: the scheduler will make sure newly bound pod will not break the existing inter pod affinity. So we do not need to invalidate MatchInterPodAffinity when pod added. But when a pod is deleted, existing inter pod affinity may become invalid. (e.g. this pod was preferred by some else, or vice versa).
 
         - NOTE: assumptions above **will not** stand when we implemented features like `RequiredDuringSchedulingRequiredDuringExecution`.
 
     - No `NoDiskConflict`: the newly scheduled pod fits to existing pods on this node, it will also fits to equivalence class of existing pods.
 
 - **Scope:** 
-    - The node which the pod was binded with.
+    - The node which the pod was bound with.
```
Comment (bsalamat) [inline contributors/design-proposals/scheduling/scheduler-equivalence-class.md]: Please replace the whole sentence with: "The node where the pod is bound". Do the same for the similar sentences below.

Comment (AdamDang) [reply contributors/design-proposals/scheduling/scheduler-equivalence-class.md]: Thanks! It's OK now.

**Comment by jdumars** [comment]
/lgtm
Thank you for the typo fixes

**Pull request merged**